tests/src/test/java/alluxio/client/fs/LoadMetadataIntegrationTest.java

```diff
@@ -63,7 +68,9 @@
   @ClassRule
   public static UnderFileSystemFactoryRegistryRule sUnderfilesystemfactoryregistry =
       new UnderFileSystemFactoryRegistryRule(new SleepingUnderFileSystemFactory(
-          new SleepingUnderFileSystemOptions().setExistsMs(SLEEP_MS)));
+          new SleepingUnderFileSystemOptions()
+              .setExistsMs(SLEEP_MS)
+              .setListStatusWithOptionsMs(LONG_SLEEP_MS)));
```

Comment:
why does this need `LONG_SLEEP_MS`? shouldn't the directly structure in the test have a large multiplying factor?

Reply:
Because I am trying to make sure listStatus on the ufs side is called once and only once,  I want the sleep length to dominate the task. Hence `LONG_SLEEP_MS`